### PR TITLE
Rollup-411: usdrif pay fees

### PR DESCRIPTION
**Pull request by illuque**
## What

-   Hardcode the value of USDRIF in the price feeder to always return 1.

## Why

-  Add support for USDRIF as token.

## Refs

-   [ROLLUP-411](https://rsklabs.atlassian.net/browse/ROLLUP-411)

### CODE REVIEW

File: core/bin/zksync_api/src/fee_ticker/ticker_info.rs [hunk 1]
```diff
@@ -20,6 +20,8 @@ use zksync_types::{Address, Token, TokenId, TokenLike, TokenPrice};
 // Local deps
 use crate::fee_ticker::PriceError;
 
+const TICKET_INFO_GET_LAST_TOKEN_PRICE: &str = "ticker_info.get_last_token_price";
```
Comment (antomor) [inline core/bin/zksync_api/src/fee_ticker/ticker_info.rs:23]: 💚

File: infrastructure/zk/src/run/run.ts [hunk 2]
```diff
@@ -29,7 +29,8 @@ export async function deployERC20(command: 'dev' | 'new', name?: string, symbol?
     if (command == 'dev') {
         await utils.spawn(`yarn --silent --cwd contracts deploy-erc20 add-multi '[
             { "name": "wBTC", "symbol": "wBTC", "decimals":  8, "implementation": "RevertTransferERC20" },
-            { "name": "RDOC", "symbol": "RDOC", "decimals": 18 },
+            { "name": "RDOCs", "symbol": "RDOC", "decimals": 18 },
```
Comment (jurajpiar) [inline infrastructure/zk/src/run/run.ts:32]: Is the name changed to `RDOCs` deliberately, or as an accident?

File: core/bin/zksync_api/src/fee_ticker/ticker_info.rs [hunk 3]
```diff
@@ -192,13 +194,21 @@ impl FeeTickerInfo for TickerInfo {
...
-            });
+            metrics::histogram!(TICKET_INFO_GET_LAST_TOKEN_PRICE, start.elapsed(), "type" => "RDOC");
+            return Ok(price_1_usd);
+        }
+
+        // TODO: remove hardcode for USDRIF token
+        if token.symbol == "USDRIF" {
+            metrics::histogram!(TICKET_INFO_GET_LAST_TOKEN_PRICE, start.elapsed(), "type" => "USDRIF");
+            return Ok(price_1_usd);
         }
```
Comment (jurajpiar) [inline core/bin/zksync_api/src/fee_ticker/ticker_info.rs:212]: could reduce to
```suggestion
if ["RDOC", "USDRIF"].contains(&token.symbol)  {
  metrics::histogram!(TICKET_INFO_GET_LAST_TOKEN_PRICE, start.elapsed(), "type" => token.symbol);
  return Ok(price_1_usd);
}
```

**Review by jurajpiar — APPROVED** [review]

File: infrastructure/zk/src/run/run.ts (hunk 2 above)
Comment (illuque) [reply infrastructure/zk/src/run/run.ts]: Probably a wrong `cmd+S`, fixed

**Review by antomor — APPROVED** [review]
@illuque I've tested it and LGTM, thank you!

File: core/bin/zksync_api/src/fee_ticker/ticker_info.rs (hunk 3 above)
Comment (antomor) [reply core/bin/zksync_api/src/fee_ticker/ticker_info.rs:212]: 👆🏽

Comment (illuque) [reply core/bin/zksync_api/src/fee_ticker/ticker_info.rs]: Good idea, done.

**Review by antomor — APPROVED** [review]
LGTM

**Pull request merged**